react-native/react/common-adapters/text.desktop.js

```diff
@@ -29,7 +30,8 @@ Text.propTypes = {
   small: React.PropTypes.bool,
   reversed: React.PropTypes.bool,
   children: React.PropTypes.node,
-  style: React.PropTypes.object
+  style: React.PropTypes.object,
+  onClick: React.PropTypes.func
```

Comment:
For now i'm explicitly adding props but we could discuss passing through everything. Feel a little mixed. Kinda sucks that key / etc doesn't work unless you add it...